test/EFCore.PG.Plugins.FunctionalTests/PGroongaQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,204 @@
+            {
+                var rows = ctx.PGroongaTypes.Count(r => EF.Functions.PrefixSearch(r.Tag, "pg"));
+                Assert.Equal(2, rows);
+                Assert.Contains(@"WHERE (r.""Tag"" &^ 'pg') = TRUE", Sql);
+            }
+        }
+
+        [Fact]
+        public void Operator_PrefixSearchIn_V2()
+        {
+            using (var ctx = CreateContext())
+            {
+                var rows = ctx.PGroongaTypes.Count(r => EF.Functions.PrefixSearchIn(r.Tag, new []{"pg", "gro"}));
+                Assert.Equal(3, rows);
+                Assert.Contains(@"WHERE (r.""Tag"" &^| ARRAY['pg','gro']::text[]) = TRUE", Sql);
+
+            }
+        }
+
+        #endregion Operators pgroonga_text_term_search_ops_v2
+
+        #region Support
+
+        PGroongaContext CreateContext() => Fixture.CreateContext();
+
+        void AssertSql(params string[] expected)
+            => Fixture.TestSqlLoggerFactory.AssertBaseline(expected);
+
+        void AssertContainsSql(params string[] expected)
+            => Fixture.TestSqlLoggerFactory.AssertBaseline(expected, assertOrder: false);
+
+        string Sql => Fixture.TestSqlLoggerFactory.Sql;
+
+        public class PGroongaFixture : SharedStoreFixtureBase<PGroongaContext>
+        {
+            protected override string StoreName { get; } = "PGroongaTest";
+
+            protected override IServiceCollection AddServices(IServiceCollection serviceCollection)
+                => base.AddServices(serviceCollection).AddEntityFrameworkNpgsqlPGroonga();
+
+            public override DbContextOptionsBuilder AddOptions(DbContextOptionsBuilder builder)
+            {
+                var optionsBuilder = base.AddOptions(builder);
+                new NpgsqlDbContextOptionsBuilder(optionsBuilder);
+
+                return optionsBuilder;
+            }
+
+            protected override void Seed(PGroongaContext context)
+                => PGroongaContext.Seed(context);
+
+            protected override ITestStoreFactory TestStoreFactory => NpgsqlTestStoreFactory.Instance;
+
+            public TestSqlLoggerFactory TestSqlLoggerFactory => (TestSqlLoggerFactory)ServiceProvider.GetRequiredService<ILoggerFactory>();
+        }
+
+        public class PGroongaContext : PoolableDbContext
+        {
+            public PGroongaContext(DbContextOptions<PGroongaContext> options) : base(options) {}
+
+            public DbSet<PGroongaType> PGroongaTypes { get; set; }
+
+            public static void Seed(PGroongaContext context)
+            {
+                context.Database.ExecuteSqlCommand("CREATE EXTENSION IF NOT EXISTS pgroonga");
+                context.Database.ExecuteSqlCommand("CREATE INDEX IF NOT EXISTS pgroonga_content_index ON \"PGroongaTypes\" USING pgroonga (\"Content\");");
+                context.Database.ExecuteSqlCommand("CREATE INDEX IF NOT EXISTS pgroonga_tag_index ON \"PGroongaTypes\" USING pgroonga (\"Tag\" pgroonga_text_term_search_ops_v2);");
```

Comment:
Couldn't this be configured using the normal configuration style (e.g. `HasExtensions(...)`)?